### PR TITLE
Puppet Parameter Documentation

### PR DESCRIPTION
**Pull request by pmyjavec**
Allows for easy injection parameter documentation blocks within Puppet.

@ripienaar any thoughts please?

### CODE REVIEW

**Comment by ripienaar** [comment]
isnt modern convention to use yard style docs @pmyjavec ?

**Comment by pmyjavec** [comment]
I hope so, the environment I am working it seems to maintain this old school style though! For now...

**Comment by lpil** [comment]
This snippet doesn't actually save any characters. What's the point?

**Comment by pmyjavec** [comment]
What do you mean save characters ?

I've been using it everyday saving me a bunch of time?

**Comment by lpil** [comment]
The trigger is at least 9 characters, it inserts 7 characters.

File: snippets/puppet.snippets [hunk 1]
```diff
@@ -84,6 +84,10 @@ snippet defheader
 		${4}
 	}
 
+snippet paramdoc
+	# [*${1:Name of parameter here}*]
+	#   ${2:Parameter description here}
```
Comment (lpil) [inline snippets/puppet.snippets]: Should this be a tab?

**Comment by pmyjavec** [comment]
Ok I understand what you mean, I have shortened the trigger and I believe the spacing is correct as per the style guide.

**Comment by pmyjavec** [comment]
No response so closing...

**Pull request closed without merging**